packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -142,7 +145,10 @@ export default function getPlugins({ languageRenderers }) {
 		getApiProxyPlugin(),
 		getLanguagePlugin(),
 		getLogger(),
-		getCsrfPlugin(),
+		getCsrfPlugin({
```

Comment:
why are we passing this?

Reply:
in `getCsrfPlugin` its not expecting any params unless i'm looking at the wrong function

Reply:
good q

Reply:
thanks, missed that - updated.